website/docs/developer/research/research-localstack.md

```diff
@@ -0,0 +1,75 @@
+---
+id: research-localstack
+title: Localstack Research Notes
+description: Research Notes on Localstack.
+---
+
+## Overview
+
+[Localstack](https://github.com/localstack/localstack) is a method of simulating an AWS environment locally.
+This can be used to bypass the requirement of a remote sandbox environment, making testing easier.
+
+
+### Implementation Details
+- Some features and not present, and some are locked behind the [Pro Version](https://localstack.cloud/#pricing).
+  - RDS is not present, but testing can be done with a local database.
```

Comment:
also ECS is not present.

Reply:
According to https://github.com/localstack/localstack, it is implemented but locked behind the 'Pro version'.